src/dataframerow/dataframerow.jl

```diff
@@ -205,12 +205,28 @@ Base.get(f::Base.Callable, dfr::DataFrameRow, key::ColumnIndex) =
 Base.broadcastable(::DataFrameRow) =
     throw(ArgumentError("broadcasting over `DataFrameRow`s is reserved"))
 
+Base.NamedTuple(dfr::DataFrameRow) = NamedTuple{Tuple(keys(r))}(values(r))
+
 """
     copy(dfr::DataFrameRow)
 
-Convert a [`DataFrameRow`](@ref) to a `NamedTuple`.
+Constucts a `NamedTuple` with the same content as the [`DataFrameRow`](@ref).
+A  [`DataFrameRow`](@ref) is a view into a row of a  [`DataFrame`](@ref), following
```

Comment:
It seems weird to say what is a `DataFrameRow` here. I'm not convinced we need to justify the behavior choice, but if you really want to, maybe this?
```
This method returns a `NamedTuple` so that the returned object
is not affected by changes to the parent data frame of which `dfr` is a view.
```

Reply:
@oxinabox felt that it is worth to justify it. I will update the wording to use your proposal.

Reply:
done